aten/src/ATen/core/jit_type.h

```diff
@@ -1535,6 +1565,8 @@ inline TypePtr TensorType::fromBoolType() {
 inline c10::optional<c10::ScalarType> tryScalarTypeFromJitType(const c10::TypePtr & type) {
   if (type == FloatType::get()) {
     return at::typeMetaToScalarType(c10::get_default_dtype());
+  } else if (type == ComplexDoubleType::get()) {
+    return at::typeMetaToScalarType(c10::get_default_complex_dtype());
```

Comment:
I don't think this change is necessary for this PR. I don't think it is being tested either.

Reply:
makes sense removed